iOS Client/Store/MBStore.swift

```diff
@@ -96,51 +118,47 @@ class MBStore: NSObject {
         performDownload(clientFunction: client.getArticlesWithCompletion, persistentContainer: persistentContainer, deserializeFunc: MBArticle.deserialize, completion: completion)
     }
     
-    private func loadDevotions(completion: @escaping ([MBDevotion]?, Error?) -> Void) {
-        
+    func saveDevotions(devotions: [LoadedDevotion]) throws {
+        try self.save(devotions, "devotions")
+    }
+    
+    private func read<T: Codable>(fromPath: String, _ resource: T.Type) throws -> T {
+        let (manager, path, _) = try urlPackage(forPath: "\(fromPath)")
+        if let data = manager.contents(atPath: path) {
+            return try parse(data, T.self)
+        } else {
+            throw StoreError.readError(msg: "No data could be read from file \(fromPath)")
+        }
+    }
+    
+    private func urlPackage(forPath: String) throws -> (FileManager, String, URL) {
```

Comment:
same thing here, could expose forPath to callers and name the parameter just path for internal usuage